slu/slu/utils/config.py

```diff
@@ -172,6 +174,132 @@ def generate(self) -> Dict[str, Config]:
         return {config_dict[const.MODEL_NAME]: config}
 
 
+class YAMLPromptConfig(ConfigDataProviderInterface):
+    """
+    An instance of this class will load, validate and fetch the state <-> prompts mapping from prompts.yaml
+    The instance will be further passed to classifier plugins like xlmr.py.
+    """
+
+    def __init__(
+        self, config_path: Optional[str] = None, debug: Optional[bool] = True
+    ) -> None:
+        self.config_path: str = config_path or const.PROMPTS_CONFIG_PATH
+        self.null_prompt_token: str = "<pad>"
```

Comment:
Please make this configurable via constructor. It can have a default value as you have right now.

Reply:
Made it configurable via constructor